private_gpt/server/completions/completions_router.py

```diff
@@ -67,6 +72,7 @@ def prompt_completion(
     message = OpenAIMessage(content=body.prompt, role="user")
     chat_body = ChatBody(
         messages=[message],
+        system_prompt=body.system_prompt,
```

Comment:
Why not simply adding a message with the role `system` if a system prompt is passed? This remove the need for this extra parameter